products/bigquery/api.yaml

```diff
@@ -114,9 +116,29 @@ objects:
         name: 'defaultTableExpirationMs'
         description: |
           The default lifetime of all tables in the dataset, in milliseconds
+      - !ruby/object:Api::Type::Integer
```

Comment:
for defaultTableExpirationMs, there's some extra stuff in the current docs that I think probably makes sense to  include here as well. Mind adding it?

Reply:
Done.